.murdock

```diff
@@ -34,7 +34,14 @@ get_apps() {
 # Only print for boards in $BOARDS.
 get_supported_boards() {
     local appdir=$1
-    for board in $(make --no-print-directory -C$appdir info-boards-supported 2>/dev/null )
+    local boards="$(make --no-print-directory -C$appdir info-boards-supported 2>/dev/null || echo broken)"
+
+    if [ "$boards" = broken ]; then
+        echo "makefile_broken"
```

Comment:
Why can't you put out the actual error message? E.g. by piping it to a temporary file or file descriptor instead of `/dev/null` above?

Reply:
Ah I see that the output should go below, but regardless I think it is helpful to print the actual error message.